packages/builder/src/actions/roundApplication.ts

```diff
@@ -196,11 +196,7 @@ const applyToRound =
       })
       .on("transactionStatus", async (res) => {
```

Comment:
should we add this step to the modal and not skip it?

Reply:
i agree with jason we should update the UI and do this properly